packages/e2e/README.md

```diff
@@ -116,38 +116,53 @@ To execute the test:
 ```
 
-## Local Network Faucet
+## Local Network
 
-The end-to-end faucet test are meant to showcase the use of the private testnet. The faucet end-to-end test shows how we can fund wallets with private testnet tADA so we can run the end-to-end tests. For the faucet end-to-end test to run, we must first start our private testnet environment as follows:
+The end-to-end local network test are meant to showcase the use of the local test network. The local network end-to-end test shows how we can fund wallets with local test network tADA so we can run the end-to-end tests. For the local network end-to-end test to run, we must first start our local test network environment as follows:
 
 ```bash
-$ yarn workspace @cardano-sdk/e2e private-network:up
+$ yarn workspace @cardano-sdk/e2e local-network:up
 ```
 
-:warning: Note that once you finish running the test, you should stop the enviroment with:
+:warning: Note that once you finish running the test, you should stop the environment with:
 
 ```bash
-$ yarn workspace @cardano-sdk/e2e private-network:down
+$ yarn workspace @cardano-sdk/e2e local-network:down
 ```
 Instead of CTRL-C, since some resources need to be clear before you can set up the environment again, if you don't stop the containers with the proper command, you may run into issues restarting it.
 
-For the faucet to work correctly, we must configure it with the mnemonic of the genesis wallet. This wallet controls all the funds on the private testnet and is the only way of obtaining tADA on that network.
+For the local network to work correctly, we must configure it with the mnemonic of the genesis wallet. This wallet controls all the funds on the local test network and is the only way of obtaining tADA on that network.
 
 ```
 # Logger
 LOGGER_MIN_SEVERITY=debug
 
 # Providers setup
-KEY_MANAGEMENT_PROVIDER=inMemory
-KEY_MANAGEMENT_PARAMS='{"accountIndex": 0, "networkId": 0, "password":"some_password","mnemonic":"run yarn workspace @cardano-sdk/e2e generate-mnemonics to generate your own"}'
 FAUCET_PROVIDER=cardano-wallet
 FAUCET_PROVIDER_PARAMS='{"url":"http://localhost:8090/v2","mnemonic":"fire method repair aware foot tray accuse brother popular olive find account sick rocket next"}'
+KEY_MANAGEMENT_PROVIDER=inMemory
+KEY_MANAGEMENT_PARAMS='{"accountIndex": 0, "networkId": 0, "password":"some_password","mnemonic":""}'
+ASSET_PROVIDER=stub
+CHAIN_HISTORY_PROVIDER=http
+CHAIN_HISTORY_PROVIDER_PARAMS='{"url":"http://localhost:4000/chain-history"}'
+NETWORK_INFO_PROVIDER=http
+NETWORK_INFO_PROVIDER_PARAMS='{"url":"http://localhost:4000/network-info"}'
+REWARDS_PROVIDER=http
+REWARDS_PROVIDER_PARAMS='{"url":"http://localhost:4000/rewards"}'
+TX_SUBMIT_PROVIDER=http
+TX_SUBMIT_PROVIDER_PARAMS='{"url":"http://localhost:4000/tx-submit"}'
+UTXO_PROVIDER=http
+UTXO_PROVIDER_PARAMS='{"url":"http://localhost:4000/utxo"}'
+STAKE_POOL_PROVIDER=stub
+STAKE_POOL_PROVIDER_PARAMS='{"url":"http://localhost:4000/stake-pool"}'
 ```
```

Comment:
```suggestion
``` How about adding a separate .env file for convenience?
```
[env.local-network.txt](https://github.com/input-output-hk/cardano-js-sdk/files/9151560/env.local-network.txt)

Reply:
addressed here https://github.com/input-output-hk/cardano-js-sdk/pull/342/commits/2680f604fb35dfd03cacca295e39af8517b499e3